gp-advanced-select/gpadvs-auto-select-on-match.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Gravity Perks // Advanced Select // Auto Select on Match
+ * https://gravitywiz.com/documentation/gravity-forms-advanced-select/
+ *
+ * Automatically select the matching option based on the search value.
+ *
+ * Instructions:
+ *
+ * 1. Install this snippet with our free Custom JavaScript plugin.
+ *    https://gravitywiz.com/gravity-forms-code-chest/
+ */
+gform.addFilter( 'gpadvs_settings', function( settings, gpadvs, selectNamespace ) {
+  if ( gpadvs.formId == GFFORMID ) {
+    settings.onType = function( str ) {
+      const tomSelect = window[ selectNamespace ];
+      const matchedKey = Object.keys( tomSelect.options ).filter( function( key ) {
+        return -1 !== tomSelect.options[ key ].text.toLowerCase().indexOf( str.toLowerCase() );
```

Comment:
I think this is a decent enough stop-gap. I am mildly curious if we can use the list of results that are fuzzy-matched as the fuzzy-match might not always match this search method. Don't worry about digging into it right now, I'm putting this note here for future reference.